autoload/maktaba/ui/selector.vim

```diff
@@ -362,8 +362,8 @@ function! maktaba#ui#selector#DoToggleHelp() dict abort
   let l:len_help = len(self._GetHelpLines())
   let self._is_verbose = !self._is_verbose
   call maktaba#buffer#Overwrite(1, l:len_help, self._GetHelpLines())
-  let &readonly = l:prev_read
-  let &modifiable = l:prev_mod
+  let &l:readonly = l:prev_read
```

Comment:
`readonly` and `modifiable` are buffer-local anyway, aren't they? (albeit I'm not quite following what `l:readonly` means in that case; is it just a redundant prefix?)

I'm also not entirely sure why this code needs to save and restore these two options, since I would have expected them to always be set the same way by the code on L279-280 above. Is there some case in which they wouldn't be set like that? (If not, we could just `setlocal noreadonly` ... `setlocal readonly`.)

Reply:
Good point.  Should I then rewrite this without saving initial options?

Reply:
:+1: if you're up for it, please do!

Reply:
I think this is definitely okay as-is, but if you can rewrite it to be clearer then :+1:.

Reply:
Done.